denoise/README.md

```diff
@@ -1,14 +1,70 @@
-# Denoising
-Various denoise algorthms implemented for Tensorflow
```

Comment:
Please create a directory for your algorithm and move files there. It currently overwrites the denoise Readme.